src/rsa/signing.rs

```diff
@@ -386,6 +386,70 @@ impl RSAKeyPair {
     }
 }
 
+
+
+/// An RSA key pair, used for signing. Feature: `rsa_signing`.
+pub struct RSAKeyPairComponents<'a> {
+    /// The public composite modulus (big endian).
+    pub n: untrusted::Input<'a>,
+    /// The public exponent (big endian).
+    pub e: untrusted::Input<'a>,
+    /// The secret exponent (big endian).
+    pub d: untrusted::Input<'a>,
+    /// The inverse of q modulo p (big endian).
+    pub iqmp: untrusted::Input<'a>,
+    /// Prime factor of n (big endian).
+    pub p: untrusted::Input<'a>,
+    /// Prime factor of n (big endian).
+    pub q: untrusted::Input<'a>,
```

Comment:
This is missing dP and dQ. Also, make sure these are all in the same order as they're given in the RFC.